cmd/orb-server/startcmd/start.go

```diff
@@ -220,13 +220,14 @@ func startOrbServices(parameters *orbParameters) error {
 	}
 
 	opProcessor := processor.New(parameters.didNamespace, opStore, pc)
+	proofHandler := proofs.New(&proofs.Providers{}, vcCh, parameters.hostURL)
```

Comment:
I think you should use the variable, apServiceIRI (line 264), which is already set to the service URI, e.g. "https://example.com/services/orb". This way you only need to append "/witnesses", which I will define as a constant when I get to the 'offer' endpoint.